XP_Pen_Artist10s/descriptors.txt

```diff
@@ -0,0 +1,120 @@
+
+Bus 003 Device 002: ID 5543:004a UC-Logic Technology Corp. 
+Device Descriptor:
+  bLength                18
+  bDescriptorType         1
+  bcdUSB               1.10
+  bDeviceClass            0 
+  bDeviceSubClass         0 
+  bDeviceProtocol         0 
+  bMaxPacketSize0        64
+  idVendor           0x5543 UC-Logic Technology Corp.
+  idProduct          0x004a 
+  bcdDevice            0.00
+  iManufacturer           5 UGEE
+  iProduct                6 10.1 Tablet Monitor
```

Comment:
This still uses the UC-Logic VID, "UGEE" iManufacturer, and is being sold as "XP-Pen". What a mess. Unless we can find the original Ugee model name this is a re-branding of, then I think we should call it by its XP-Pen name, as you did. However, please note the spelling: "XP-Pen", not "XP Pen", and please name the directory accordingly. I.e. `XP-Pen_Artist_10S`.

In general, please use manufacturer's name, spelling, and capitalization.